server.js

```diff
@@ -1,14 +1,18 @@
 const express = require("express");
 const cors = require("cors");
+const { json } = require("express");
```

Comment:
As discussed on the call - this is not required as we use `express.json()` later on